apisix/discovery/consul_kv.lua

```diff
@@ -356,6 +434,20 @@ function _M.init_worker()
         return
     end
 
+    if consul_conf.dump then
+      local dump = consul_conf.dump
+      if not dump.path or #dump.path == 0 then
```

Comment:
We can use `required` & `minLength` to validate the path in the schema.

Reply:
Y